src/metabase/driver/sql/query_processor.clj

```diff
@@ -503,22 +571,35 @@
     (hx/call :count (->honeysql driver field))
     :%count.*))
 
-(defmethod ->honeysql [:sql :avg]        [driver [_ field]]   (hx/call :avg             (->honeysql driver field)))
-(defmethod ->honeysql [:sql :median]     [driver [_ field]]   (hx/call :median          (->honeysql driver field)))
-(defmethod ->honeysql [:sql :percentile] [driver [_ field p]] (hx/call :percentile-cont (->honeysql driver field) (->honeysql driver p)))
-(defmethod ->honeysql [:sql :distinct]   [driver [_ field]]   (hx/call :distinct-count  (->honeysql driver field)))
-(defmethod ->honeysql [:sql :stddev]     [driver [_ field]]   (hx/call :stddev_pop      (->honeysql driver field)))
-(defmethod ->honeysql [:sql :var]        [driver [_ field]]   (hx/call :var_pop         (->honeysql driver field)))
-(defmethod ->honeysql [:sql :sum]        [driver [_ field]]   (hx/call :sum             (->honeysql driver field)))
-(defmethod ->honeysql [:sql :min]        [driver [_ field]]   (hx/call :min             (->honeysql driver field)))
-(defmethod ->honeysql [:sql :max]        [driver [_ field]]   (hx/call :max             (->honeysql driver field)))
+(defmethod ->honeysql [:sql :avg]    [driver [_ field]] (hx/call :avg        (->honeysql driver field)))
```

Comment:
Why don't we have to do a version switch on all the `hx/*` usages in this ns? These `->honeysql` and the `date` (line 130) defmethods especially.

Reply:
I moved the original `hx` stuff to `m.u.honey-sql-1` a while back. The new version of `hx` will hand off to either `m.u.honey-sql-1` or `m.u.honey-sql-2` depending on `*honey-sql-version*`. So it works with either version